src/librustdoc/html/static/rustdoc.css

```diff
@@ -1141,36 +1141,53 @@ kbd {
 	border: solid 1px;
 	border-radius: 3px;
 	box-shadow: inset 0 -1px 0;
+	cursor: default;
 }
 
-#theme-picker {
+.theme-picker {
 	position: absolute;
 	left: 211px;
-	top: 17px;
+	top: 19px;
+}
+
+#theme-picker {
 	padding: 4px;
+	width: 27px;
+	height: 29px;
 	border: 1px solid;
 	border-radius: 3px;
 	cursor: pointer;
 }
 
+#theme-picker:hover {
+	border-color: #717171;
+}
+
 #theme-choices {
 	display: none;
 	position: absolute;
-	left: -1px;
-	top: 30px;
+	left: 0;
+	top: 28px;
 	border: 1px solid;
 	border-radius: 3px;
 	z-index: 1;
+	cursor: pointer;
 }
 
-#theme-choices > div {
-	border-top: 1px solid;
+#theme-choices > button {
+	border: none;
+	width: 100%;
 	padding: 4px;
 	text-align: center;
+	background: rgba(0,0,0,0);
```

Comment:
Pretty sure there's a conversation somewhere about where the colors need to go, since you recently made sure every color directive went into the theme css. Or did you want the theme-picker button to look the same in every theme? `>_>`

Reply:
I want to the background of the button to be invisible so that the buttons background is consistent to its parent. It's the only case where I'll put colors in this file.

Reply:
Ohhhh, no alpha, now i get it.

Reply:
Hehe.